data/json/items/gunmod/muzzle.json

```diff
@@ -70,15 +69,14 @@
     "color": "dark_gray",
     "location": "muzzle",
     "mod_targets": [ "shotgun" ],
-    "damage_modifier": -4,
-    "loudness_modifier": -30,
-    "recoil_modifier": -15
+    "loudness_modifier": -50,
+    "recoil_modifier": -25
```

Comment:
I'm ambivalent towards `recoil` (noting IRL @Rivet-the-Zombie is correct). Suggestions as to `loudness`?